server_test.go

```diff
@@ -1440,7 +1476,7 @@ func TestServer_RawDataReturnsInOrder(t *testing.T) {
 		t.Fatalf("unexpected error during GROUP BY *: %s", res.Err)
 	} else if len(res.Series) != 10 {
 		t.Fatalf("expected 10 series back but got %d", len(res.Series))
-	} else if len(res.Series[0].Values) != 50 {
+	} else if len(res.Series[1].Values) != 50 {
```

Comment:
ordering changed so the one series that has 49 values happened to be in the first position.